scripts/make_header.sh

```diff
@@ -8,7 +8,7 @@ namespace esphome {
 namespace $3 {
 
 EOT
-echo "const uint8_t INDEX_GZ[] PROGMEM = {" >> ./$1/$2
+echo "const uint8_t $4[] PROGMEM = {" >> ./$1/$2
```

Comment:
@afarago Was this needed for the vfallback work and can this potentially be removed?

Reply:
I would think yes, lets revert this change for this PR